cmd/get/get_urls.go

```diff
@@ -0,0 +1,192 @@
+package get
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/splicemachine/splicectl/common"
+	netw "k8s.io/api/networking/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type namedURL struct {
+	name, url string
+}
+
+const (
+	oauthProxySuffix = "-oauth2-proxy"
+	ssNameSpace      = "splice-system"
+	displayNameLabel = "displayName"
+	defaultNameLabel = "app"
+)
+
+var getUrlsCmd = &cobra.Command{
```

Comment:
Wasn't it included in another PR? 
Was it already merged?

Reply:
[This one](https://splicemachine.atlassian.net/browse/DBAAS-4212?atlOrigin=eyJpIjoiODlkMzNkMTRkY2QwNDQ2NmFmZWYwOGRlZTZhNGRiNGUiLCJwIjoiaiJ9) has not yet been merged. They are both being developed at the same time, so I branched this one off of get urls as they shared some functionality.

Reply:
I just merged the URL PR, it doesn't show as a merge conflict, so it looks like it will be good.